variables.tf

```diff
@@ -144,6 +144,12 @@ variable "domain_name" {
   default     = ""
 }
 
+variable "domain_name_zone_name" {
```

Comment:
I'm not super happy with the name of the variable, but tried to stay close to the name of other variables in that context (e.g. `domain_name_certificate_arn`).
Maybe just `hosted_zone_name` or `domain_hosted_zone_name` would be better?

Reply:
The name is okay, but the description can be updated a bit.

Reply:
should it be called `apex_domain_name` or `base_domain_name`?

Reply:
I agree that `apex_domain_name` is a much better name.

Reply:
Sounds good to me 🙂

Reply:
From what I've been reading the `apex` domain would be for example `acme.com`, while the value we pass here might also be a subdomain that's been delegated to our account. Since we're on AWS maybe it would make sense to name it the same way AWS does and go for `hosted_zone_name` after all?

Reply:
I vote for `hosted_zone_name`, if we are voting for names :)

Reply:
Alright, updated! 🙂